corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -208,7 +208,7 @@ hqDefine("cloudcare/js/form_entry/form_ui", [
          * @param {Object} child - The child object to be rendered, either Group, Repeat, or Question
          */
         self.childTemplate = function (child) {
-            return ko.utils.unwrapObservable(child.type) + '-fullform-ko-template';
+            return ko.utils.unwrapObservable(child.type()) + '-fullform-ko-template';
```

Comment:
I think you can remove the `ko.utils.unwrapObservable`, since `child.type()` will never return an observable, right?

On the other hand, when I see `ko.utils.unwrapObservable` I assume the code is intentionally prepared to handle either an observable or a primitive. How confident are you that `type` is never a primitive?